app/scripts/controllers/reviewRegistration.js

```diff
@@ -68,6 +68,105 @@ angular.module('confRegistrationWebApp')
       return _.find($scope.blocks, {id: blockId});
     };
 
+    var setRegistrationAsCompleted = function() {
+      registration = angular.copy(registration);
+      registration.completed = true;
+
+      RegistrationCache.update('registrations/' + registration.id, registration, function () {
+        RegistrationCache.emptyCache();
+        if(conference.registrationCompleteRedirect){
+          $window.location.href = conference.registrationCompleteRedirect;
+        }else{
+          $route.reload();
+        }
+      }, function (data) {
+        $scope.currentRegistration.completed = false;
+        $scope.submittingRegistration = false;
+        modalMessage.error(data.error ? data.error.message : 'An error occurred while submitting your registration.');
+      });
+    };
+
+    ////// START EVENT-433 //////
+
+    //Use $http.get using current registrant to get endPointResult and endPointSpouseInfo
+    var endPointResult = 200;
+    var endPointSpouseInfo = {firstName: 'Jane', lastName: 'Worner', email: 'janeworner@outlook.com'};
+
+    //If get 200 back from endpoint then show Include Spouse button
+    //else if 204 do not show button
+    if(endPointResult === 200) {
```

Comment:
To determine whether or not the to show the button, I think we need to use Lee's `getSpouseRegistration` API endpoint. If the spouse registration exists and includes the current user, then warn the user that they are already registered. If the spouse registration exists and does not include the current user, show a button that lets the user combine their pending registration with their spouse's existing registration. If the spouse registration does not exist, just show the normal registration confirmation button.